docs/whats-new.md

```diff
@@ -46,7 +46,7 @@ Date: Apr 24, 2018
 
 ## Transform class (New, WebGL2) (Experimental)
 
-The new experimental [`Transform`](/docs/api-reference/core/experimental/transform.md) class provides an easy-to-use interface to perform Transform Feedback operations.
+The new experimental [`Transform`](/docs/api-reference/core/transform.md) class provides an easy-to-use interface to perform Transform Feedback operations.
```

Comment:
Can you move `transform.md` file, otherwise this link will not work.

Reply:
> Can you move transform.md file, otherwise this link will not work.

I checked and tested the website, looks like it was already moved...

Reply:
It will still work since the content is served from last release branch , but once we release a branch off master, it might not work.